tests/app/questionnaire/test_questionnaire_store_updater.py

```diff
@@ -446,7 +450,7 @@ def test_update_relationship_question_completeness_no_relationship_collectors(
     )
 
     assert (
-        questionnaire_store_updater.update_relationship_question_completeness(
+        questionnaire_store_updater._update_relationship_question_completeness(  # pylint: disable=protected-access
```

Comment:
what was wrong with it being public?

Reply:
See this comment from @MebinAbraham - https://github.com/ONSdigital/eq-questionnaire-runner/pull/1044#discussion_r1196127989

Reply:
isn't the comment about reverting them to public?

Reply:
Oh I see I thought you were referring to that one. The ones that I updated that are not being called directly in tests, were to avoid the `too-many-public-methods` linting error

Reply:
Fair enough, guess either way requires disabling an error